external/libucl/Makefile.autosetup

```diff
@@ -17,4 +17,11 @@ LOCAL_CFLAGS=	-I$(top_srcdir)/external/libucl/include \
 		-Wno-unused-parameter \
 		-Wno-pointer-sign
 
+VPATH=$(top_srcdir)/external/libucl
+
+mkoutdir:
+	mkdir -p $(top_builddir)/external/libucl/src
+
+all: mkoutdir
```

Comment:
Do these parallelise correctly?..

Reply:
Probably not, no.  Perhaps should happen as part of `./configure`?